src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -497,7 +497,7 @@ private boolean perform(Run src, Run<?,?> dst, String expandedFilter, String exp
     public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
 
         public FormValidation doCheckProjectName(
-                @AncestorInPath AbstractProject anc, @QueryParameter String value) {
+                @AncestorInPath Job<?,?> anc, @QueryParameter String value) {
```

Comment:
I think you're talking about the latter issue described in [JENKINS-28096](https://issues.jenkins-ci.org/browse/JENKINS-28096).
That's already fixed with your change #62 . Thanks.
I updated [JENKINS-28096](https://issues.jenkins-ci.org/browse/JENKINS-28096).